corehq/apps/api/urls.py

```diff
@@ -11,35 +11,11 @@
 from django.conf.urls import *
 from django.http import HttpResponseNotFound
 from tastypie.api import Api
-from corehq.apps.api.es import XFormES
 
 
 API_LIST = (
-    ((0, 1), (
-        v0_1.CommCareUserResource,
-        v0_1.WebUserResource,
-        v0_1.CommCareCaseResource,
-        v0_1.XFormInstanceResource,
-        FixtureResource,
-        DomainMetadataResource,
-    )),
-    ((0, 2), (
-        v0_1.CommCareUserResource,
-        v0_1.WebUserResource,
-        v0_2.CommCareCaseResource,
-        v0_1.XFormInstanceResource,
-        FixtureResource,
-        DomainMetadataResource,
-    )),
     ((0, 3), (
-        v0_1.CommCareUserResource,
-        v0_1.WebUserResource,
         v0_3.CommCareCaseResource,
```

Comment:
This is still used by 2 project (but only a few requests). We could try and communicate out to them that we're removing this API and they should use v0.4 or v0.5.